### ISSUE TITLE
Balancing USA powerplants

### ISSUE THREAD

**Issue by ExiLe-ZH** [TheSuperHackers/GeneralsGamePatch#1503]
| Faction | Unit Cost | Upgrade Cost | Power | Upgraded Power |
|---------|----------:|-------------:|------:|---------------:|
| USA     | 800       | 500          | 5     | 10 (5+5)             |
| AFG     | 800       | 500          | 5     | 10 (5+5)             |
| Laser   | 700       | 500          | 8     | 16 (8+8)             |
| SWG     | 900       | 500          | 5     | 20 (5+15)             |

It doesn't seem entirely fair for SWG, her base reactor costs more while not generating extra power. Ofcourse Laser needs the extra power for his Laser Crusader production, but SWG doesn't have this option at all. She relies alot on her defenses since she got more expensive Vees aswell. Furthermore Laser is allowed to make a Laser Patriot on two Supplies without needing the Control Rods Upgrade, while SWG can not.

### Proposals

1. Increase SWG base power production (8 or 10).
2. Reduce SWG Reactor price (optionally increase Upgrade price to compensate).

**Comment by xezon** [comment]
I do like the existing composition. The upgraded SWG Power Plant is very good.

**Comment by ExiLe-ZH** [comment]
Upgraded SWG reactor is good, but sucks without upgrade, while being the most expensive one. SWG is always forced to upgrade (invest double, $1400 vs $700), while Laser can make (or drop) a Laserpat without needing the upgrade (with your standard Dual Sup WF BO). Keeping in mind that SWG is forced to play more defensively with her slower unit production, it doesn't make a whole lot of sense.

**Comment by ExiLe-ZH** [comment]
We can actually see in practice that the current model is often problematic for SWG. When SWG loses an upgraded reactor (in early games), it's very hard to recover, because it takes a long time rebuild AND reupgrade the Reactor.

**Comment by xezon** [comment]
I would say that 5+15 power is much better than 5+5, even if it costs 100$ more. In essence, SWG gets the better Power Deal.

**Comment by ExiLe-ZH** [comment]
Except that USA/AFG (and again, Laser) don't need the upgrade at all early game , they can just Vee spam and build a Firebase/Laserpat when needed. Ofcourse SW can also make Firebases, but she is supposed to have a defensive advantage with EMP's. 

I think SW would be quite a bit stronger with better power management.

**Comment by xezon** [comment]
Maybe make it 8+12

**Comment by MTKing4** [comment]
8+12 sounds good

Exiles point makes sense to me, SWG doesn't really have good options early game

**Comment by ExiLe-ZH** [comment]
I'd like to try and test this, anyone up for it this weekend?

**Comment by MTKing4** [comment]
I might be able to test with you during weekdays, currently my weekends are occupied

**Comment by Stubbjax** [comment]
8+12 feels like a big change that would affect / change the early-game dynamic too much from 1.04. A $100 reduction from the initial cost feels a lot safer. It could also be an idea to reduce Control Rods research time, e.g. from 30s to 25s, or the price, e.g. from $500 to $400 for an even 50% split (wait three times as long for half the price).

**Comment by ExiLe-ZH** [comment]
I like the idea of improved power management much better than (Vee) price reductions for SW. It would be boring if SW can play more like other USA's where games are won by spam and early pushes. Laser has great power management AND spam, so I don't feel it would be too much of a buff, but it will help SW alot in setting up defensive perimeters and being much less fragile before that.

**Comment by xezon** [comment]
Close?

**Comment by Stubbjax** [comment]
Has the issue been solved?

**Comment by ExiLe-ZH** [comment]
To me sounds like the most optimal starting point of buffing SW while keeping the faction stay true to the originally intended pros and cons.

**Comment by Stubbjax** [comment]
Reducing the initial cost by $100 seems fair, and would help offset the initial $850 Humvee cost without impacting the feel of the game. This would reduce the total cost per unit of power from $70 to $65, which is a more meaningful gap between Laser General's $75 per unit of power.

Laser General's advanced power is not highlighted anywhere in-game, so it seems counterintuitive that the values are so close.

**Comment by xezon** [comment]
When base price decreases from 900 to 800, should upgrade price increase from 500 to 600?

**Comment by Stubbjax** [comment]
Well the nice thing about a $100 reduction is that it adds consistency with vUSA and Air Force reactors, and widens the gap with Laser. Increasing the upgrade price would cancel out both the bonus consistency and gap.

**Comment by xezon** [comment]
So no?

**Comment by Stubbjax** [comment]
Correct. It wouldn't really help at all with the point Exile described [here](https://github.com/TheSuperHackers/GeneralsGamePatch/issues/1503#issuecomment-1353353088). A $100 reduction however, while not a direct solution, would at least reduce the severity.

**Comment by xezon** [comment]
Reference:

* TheSuperHackers/GeneralsGameCode#139

Is the Laser General Power Plant Energy level sufficient? It gives +3 +3 power, but Laser Turrets (-5) and Tanks (-1) will deduct 2 and 1 extra power compared to other USA.

| Object              | Unit Cost | Upgrade Cost | Power | Upgraded Power |
|---------------------|----------:|-------------:|------:|---------------:|
| USA Power Plant     | 800       | 500          | 5     | 10 (5+5)       |
| AFG Power Plant     | 800       | 500          | 5     | 10 (5+5)       |
| Laser Power Plant   | 700       | 500          | 8     | 16 (8+8)       |
| SWG Power Plant     | 900       | 500          | 5     | 20 (5+15)      |

| Object         | Power Consumption |
|----------------|------------------:|
| USA Patriot    | -3                |
| AFG Patriot    | -3                |
| Laser Patriot  | -5                |
| Laser Tank     | -1                |
| SWG Patriot    | -3                |

**Comment by ExiLe-ZH** [comment]
The design isn't exactly set up as I thought it was.

In terms of defense spam it actually scales worse:
- One Laser Patriot occupies a Reactor by 62,5% (5/8).
- One Vanilla Patriot occupies a Vanilla Reactor by 60% (3/5).

However it does scale better with the other structures (Supply Centers, War Factories, Airfields, Strategy Center, Supply Dropzones, Particle Cannons) as their requirements were not scaled up.
